### PR TITLE
Maui: Updated UI in UtilityNetwork category 

### PR DESCRIPTION
**Pull request by amweis226**
# Description

Remove response form container from UtilityNetwork samples and fix bugs associated with porting to MAUI

## Type of change

- [x] Bug fix
- [x] Sample viewer enhancement

**Platforms tested on**:

- [x] MAUI Windows
- [x] MAUI Android
- [x] MAUI iOS

## Checklist

- [x] Runs and compiles on all active platforms
- [ ] Legacy platforms still compile and run (if applicable)
- [x] Branch is up to date with the latest main/v.next
- [x] All merge conflicts have been resolved
- [x] Self-review of changes
- [x] There are no warnings related to changes
- [x] No unrelated changes have been made to any other code or project files
- [ ] `sample_sync.py` runs without making changes
- [x] Codemaid and XAML styler extensions have been run on every changed file
- [x] Code is commented with correct formatting
- [x] All variable and method names are good and make sense
- [x] There is no leftover commented code
- [ ] Screenshots are correct size and display in description tab

### CODE REVIEW

File: src/MAUI/Maui.Samples/Samples/UtilityNetwork/ConfigureSubnetworkTrace/ConfigureSubnetworkTrace.xaml
Comment (duffh) [inline src/MAUI/Maui.Samples/Samples/UtilityNetwork/ConfigureSubnetworkTrace/ConfigureSubnetworkTrace.xaml:50]: I'd stack these pickers vertically and give them a minimum width as they render very small on android. The entry is hard to see if the first two pickers have values.

File: src/MAUI/Maui.Samples/Samples/UtilityNetwork/ConfigureSubnetworkTrace/ConfigureSubnetworkTrace.xaml
Comment (duffh) [inline src/MAUI/Maui.Samples/Samples/UtilityNetwork/ConfigureSubnetworkTrace/ConfigureSubnetworkTrace.xaml:43]: Potentially put borders around the condition section and each of the other sections to clearly indicate which part of the control is interacting with each button or output.

File: src/MAUI/Maui.Samples/Samples/UtilityNetwork/ConfigureSubnetworkTrace/ConfigureSubnetworkTrace.xaml
Comment (duffh) [inline src/MAUI/Maui.Samples/Samples/UtilityNetwork/ConfigureSubnetworkTrace/ConfigureSubnetworkTrace.xaml:76]: I think the `FontSize="Title"` is a little large when compared to the other font sizes in this sample.

File: src/MAUI/Maui.Samples/Samples/UtilityNetwork/ConfigureSubnetworkTrace/ConfigureSubnetworkTrace.xaml
Comment (duffh) [inline src/MAUI/Maui.Samples/Samples/UtilityNetwork/ConfigureSubnetworkTrace/ConfigureSubnetworkTrace.xaml:5]: A maximum width might be useful here as the UI becomes very stretched in horizontal orientation on mobile.

File: src/MAUI/Maui.Samples/Samples/UtilityNetwork/DisplayUtilityNetworkContainer/DisplayUtilityNetworkContainer.xaml
Comment (duffh) [inline src/MAUI/Maui.Samples/Samples/UtilityNetwork/DisplayUtilityNetworkContainer/DisplayUtilityNetworkContainer.xaml:57]: I think it might be a nicer user experience to enable/disable the button rather than showing/hiding.

File: src/MAUI/Maui.Samples/Samples/UtilityNetwork/DisplayUtilityNetworkContainer/DisplayUtilityNetworkContainer.xaml
Comment (duffh) [inline src/MAUI/Maui.Samples/Samples/UtilityNetwork/DisplayUtilityNetworkContainer/DisplayUtilityNetworkContainer.xaml:18]: Column spacing would be nice between the image and the label items.

File: src/MAUI/Maui.Samples/Samples/UtilityNetwork/DisplayUtilityAssociations/DisplayUtilityAssociations.xaml
Comment (duffh) [inline src/MAUI/Maui.Samples/Samples/UtilityNetwork/DisplayUtilityAssociations/DisplayUtilityAssociations.xaml:14]: Column spacing would be good here.

File: src/MAUI/Maui.Samples/Samples/UtilityNetwork/PerformValveIsolationTrace/PerformValveIsolationTrace.xaml.cs
Comment (duffh) [inline src/MAUI/Maui.Samples/Samples/UtilityNetwork/PerformValveIsolationTrace/PerformValveIsolationTrace.xaml.cs:276]: I'm not sure that we just want to catch and consume this here?

File: src/MAUI/Maui.Samples/Samples/UtilityNetwork/TraceUtilityNetwork/TraceUtilityNetwork.xaml
Comment (duffh) [inline src/MAUI/Maui.Samples/Samples/UtilityNetwork/TraceUtilityNetwork/TraceUtilityNetwork.xaml:91]: Need to remove the text color for dark more/light mode themes.

File: src/MAUI/Maui.Samples/Samples/UtilityNetwork/TraceUtilityNetwork/TraceUtilityNetwork.xaml
Comment (duffh) [inline src/MAUI/Maui.Samples/Samples/UtilityNetwork/TraceUtilityNetwork/TraceUtilityNetwork.xaml:42]: Width changes when button text/label text changes in horizontal orientation. Fixed/max width would prevent this.

**Review by duffh — CHANGES_REQUESTED** [review]
Looks good, just a few nitpicking comments.

File: src/MAUI/Maui.Samples/Samples/UtilityNetwork/TraceUtilityNetwork/TraceUtilityNetwork.xaml
Comment (ZackAllen) [inline src/MAUI/Maui.Samples/Samples/UtilityNetwork/TraceUtilityNetwork/TraceUtilityNetwork.xaml:43]: Padding of 5 here will prevent button cropping on WinUI.

File: src/MAUI/Maui.Samples/Samples/UtilityNetwork/TraceUtilityNetwork/TraceUtilityNetwork.xaml
Comment (ZackAllen) [inline src/MAUI/Maui.Samples/Samples/UtilityNetwork/TraceUtilityNetwork/TraceUtilityNetwork.xaml:11]: This is not centered in the screen. Set ColumnSpan and RowSpan values.

File: src/MAUI/Maui.Samples/Samples/UtilityNetwork/TraceUtilityNetwork/TraceUtilityNetwork.xaml
Comment (ZackAllen) [inline src/MAUI/Maui.Samples/Samples/UtilityNetwork/TraceUtilityNetwork/TraceUtilityNetwork.xaml:24]: This is no longer being used, it should be removed.

File: src/MAUI/Maui.Samples/Samples/UtilityNetwork/ConfigureSubnetworkTrace/ConfigureSubnetworkTrace.xaml.cs
Comment (ZackAllen) [inline src/MAUI/Maui.Samples/Samples/UtilityNetwork/ConfigureSubnetworkTrace/ConfigureSubnetworkTrace.xaml.cs:71]: Commented code should be removed.

File: src/MAUI/Maui.Samples/Samples/UtilityNetwork/PerformValveIsolationTrace/PerformValveIsolationTrace.xaml
Comment (ZackAllen) [inline src/MAUI/Maui.Samples/Samples/UtilityNetwork/PerformValveIsolationTrace/PerformValveIsolationTrace.xaml:30]: This text color makes the text unreadable on MacCatalyst.

File: src/MAUI/Maui.Samples/Samples/UtilityNetwork/PerformValveIsolationTrace/PerformValveIsolationTrace.xaml.cs
Comment (amweis226) [reply src/MAUI/Maui.Samples/Samples/UtilityNetwork/PerformValveIsolationTrace/PerformValveIsolationTrace.xaml.cs:276]: This is to allow users to tap outside the bounds of the pop-up DisplayActionSheet (especially buttons in other UI elements) without triggering an unhandled invalid value exception; this way, the DisplayActionSheet can be cancelled with no further action taken

File: src/MAUI/Maui.Samples/Samples/UtilityNetwork/TraceUtilityNetwork/TraceUtilityNetwork.xaml
Comment (amweis226) [reply src/MAUI/Maui.Samples/Samples/UtilityNetwork/TraceUtilityNetwork/TraceUtilityNetwork.xaml:11]: On the StackLayout or the GeoView?

File: src/MAUI/Maui.Samples/Samples/UtilityNetwork/ConfigureSubnetworkTrace/ConfigureSubnetworkTrace.xaml
Comment (amweis226) [reply src/MAUI/Maui.Samples/Samples/UtilityNetwork/ConfigureSubnetworkTrace/ConfigureSubnetworkTrace.xaml]: Alternated element backgrounds into theme-appropriate colors

**Review by duffh — APPROVED** [review]
Approved based on the updates after the last review. Hopefully Zack can take another look before you merge.

File: src/MAUI/Maui.Samples/Samples/UtilityNetwork/ConfigureSubnetworkTrace/ConfigureSubnetworkTrace.xaml
Comment (ZackAllen) [inline src/MAUI/Maui.Samples/Samples/UtilityNetwork/ConfigureSubnetworkTrace/ConfigureSubnetworkTrace.xaml:6]: We shouldnt use the style here, it should only be used in the full pattern.

**Review by ZackAllen — APPROVED** [review]

**Pull request merged**
